packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -137,6 +138,11 @@ exports.config = {
       ...WDIO_INTERNAL_PORT && { port: WDIO_INTERNAL_PORT },
       ...fs.existsSync(defaultWebpackPath) && { webpackConfig: defaultWebpackPath },
     }],
+    [VisualRegressionLauncher, {
+      baseScreenshotDir: process.cwd(),
```

Comment:
Do we need to maintain the baseScreenshotDir override that existed in the last version?

Reply:
Ahh...I didn't know we currently allow this to be overridden [here](https://github.com/cerner/terra-toolkit-boneyard/blob/ea4a5fc159f8e5f71911077ef613259694940e47/config/wdio/visualRegressionConf.js#L69). Looks like this is currently only used in our theme repos. We can talk during our next sync to decided whether or not to allow the override.

Reply:
We are also using this to run grid tests on Terra internally

Reply:
discussed yesterday that we don't want to implement this override. The repos that need that use-case can provide custom configuration.

Reply:
Removed configuration option for baseScreenshotDir. https://github.com/cerner/terra-toolkit/pull/545/commits/9933873194e2d7e6d395853f313cdd4cf89d1212